app/models/deploy_block.rb

```diff
@@ -3,8 +3,8 @@
 class DeployBlock < ApplicationRecord
   belongs_to :project
 
-  scope :unresolved, -> { where(resolved_at: nil) }
-  scope :resolved, -> { where.not(resolved_at: nil) }
+  scope :unresolved, -> { where(resolved_at: nil).or(where('resolved_at >= ?', DateTime.current)) }
+  scope :resolved, -> { where.not(resolved_at: nil).and(where('resolved_at <= ?', DateTime.current)) }
```

Comment:
I think we can remove the whole `...not(resolved_at: nil)` condition and just replace it with the new one.

Reply:
make sense, just removed